xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -666,6 +666,8 @@ The parameterless constructor for the <xref:System.Security.Cryptography.X509Cer
 Calling the <xref:System.Security.Cryptography.X509Certificates.X509ChainPolicy.Reset> method replaces the value of <xref:System.Security.Cryptography.X509Certificates.X509ChainPolicy.VerificationTime> with the current system time, 
 not with the time that the object was constructed.
 
+This property will not account for revocation time of the certificate on non-Windows OS-es due to platform limitations.
```

Comment:
Perhaps it would be clearer to just start the sentence on line 663 with "On Windows"?

Reply:
any specific suggestion? Not sure how to phrase that to emphasize it will only work there... The only thing which comes to mind would be something like `On Windows this property will account for revocation time of the certificate.`

Reply:
Well, it's currently "This property may also be used to determine the revocation status of a certificate.", so

"On Windows, this property may also be used to determine the revocation status of a certificate." or "This property may also be used to determine the revocation status of a certificate, on Windows only." or something to that effect.

Reply:
I like `On Windows, this property may also be used to determine the revocation status of a certificate.` although wondering if this should be `will` instead of `may`, I think `may` might imply choice